### PR TITLE
Change retire-worker logic

### PR DESCRIPTION
**Pull request by akrajewski**
Wait for graceful exit of the concourse worker process instead of looking for a certain log message (as the message can change between releases)

### CODE REVIEW

**Comment by ahelal** [comment]
cool thanks for the PR, but my initial feeling 60 second as a default timeout, might be too little. 
What are your thoughts about 60 sec ?

**Comment by akrajewski** [comment]
Thank for getting back to me :)!

In fact I haven't put that much thought into it. I've just picked a value that seemed okay i.e. was not too high compared to the previous timeout of 15 seconds per retire attempt and seemed to work fine according to quick check in my setup.

I agree though that we can make it higher if you are okay with it. The previous default configuration was to do 10 retire attempts with 15 seconds of timeout each... which gives us 150 seconds of "total timeout".  So maybe let's go with 150 seconds instead of 60? That would give us kind of two scanrios here:

1.  If `concourse retire-worker` command succeeds then we wait for 150 seconds for the process to exit
2. If `concourse retire-worker` command fails then we retry 10 times through retire attempts

The worst case is if for some reason the command is successful but then the process doesn't exit... We would then have a total timeout of 150s*10=1500s. But I guess this is really a corner case and kind of a critical error from Concourse side which would require manual intervention and troubleshooting anyway.

TL;DR I agree that the timeout can be increased to something like 150 sec. Let me know what you think and I will update the PR accordingly :)

**Comment by ahelal** [comment]
Hi @akrajewski 

Use please update to 150sec and i will merge after that

**Comment by ahelal** [comment]
👍

**Comment by akrajewski** [comment]
Thanks for merge 👍
